stubs/pyasn1/pyasn1/type/base.pyi

```diff
@@ -41,7 +41,7 @@ class NoValue:
     def __getattr__(self, attr) -> None: ...
     # def __new__.<locals>.getPlug.<locals>.plug
     @type_check_only
-    def plug(self, *args: object, **kw: object): ...
+    def plug(self, *args: Unused, **kw: Unused) -> NoReturn: ...
```

Comment:
I'm not sure I understand why this method is in the stub at all.

Reply:
Pretty sure it's just to use as an alias below. Source: https://github.com/pyasn1/pyasn1/blob/main/pyasn1/type/base.py#L212